docs/plugins/stonesense.rst

```diff
@@ -56,6 +56,16 @@ views, fog, and rotation. Here's the important section:
    :literal:
    :end-before: VALID ACTIONS:
 
+
+Streaming stonesense
+--------------------
+If you wish to stream stonesense thru OBS then you may find that opening
```

Comment:
You should explain what OBS is and provide a link. It is not a given that readers will know what that is. Also, you should write more about which operating system environment(s) this applies to, and whether it applies to other screen capturing software as well.

Reply:
how's it look now?